.pfnci/hint.pbtxt

```diff
@@ -0,0 +1,16 @@
+# hint.pbtxt is a config file for xpytest.
+#
+# Proto type: xpytest.proto.HintFile
+# https://github.com/chainer/xpytest/blob/master/proto/test_case.proto
+
+# Slow tests take 60+ seconds.
+rules { name: "agents_tests/test_ppo.py" xdist: 4 deadline: 240 }
+rules { name: "agents_tests/test_trpo.py" xdist: 4 deadline: 240 }
+
+# Slow tests take 10+ seconds.
+rules { name: "agents_tests/test_acer.py" }
+rules { name: "agents_tests/test_ale.py" }
+rules { name: "agents_tests/test_trpo.py" }
+rules { name: "policies_tests/test_deterministic_policy.py" }
+rules { name: "q_functions_tests/test_state_action_q_function.py" }
+rules { name: "tests/test_ale.py" }
```

Comment:
これってnameだけ指定してるときにどういう意味があるかわからないんですが、xpytestのどこを見ればわかりますか？

Reply:
Created a pull request in xpytest to clarify the situation: https://github.com/chainer/xpytest/pull/10

Reply:
Sorry but I still don't understand what `rules { name: "agents_tests/test_acer.py" }` does.

Reply:
This changes the order of test cases to test (the specified tests are prioritized). It enables xpytest to use CPU cores more efficiently because long lingering tests often make some CPU cores idle.

Reply:
My understanding is that tests specified in `.pfnci/hint.pbtxt` will be executed earlier than the non-specified, so tests that take long time should be specified here to utilize CPUs efficiently. That makes sense. Correct if I'm wrong.